libraries/default.rb

```diff
@@ -465,11 +478,13 @@ def download_episode_defaults_json_file(current_deploy_root)
         recursive true
       end
 
-      execute 'download the EpisodeDefaults.json file into the correct location' do
-        command %Q|cd #{episode_default_storage_dir} && aws s3 cp s3://#{private_assets_bucket_name}/EpisodeDefaults.json EpisodeDefaults.json|
-        retries 10
-        retry_delay 5
-        timeout 300
+      if node['vagrant_environment'] != true
```

Comment:
yes, this is repeated logic. I could probably figure some way of getting rid of the repetition.